### ISSUE TITLE
Version 2.20.0 broken on Xubuntu 20.04

### ISSUE THREAD

**Issue by emx** [LedgerHQ/ledger-live-desktop#3527]
#### Ledger Live Version and Operating System

- tested on Ledger Live 2.20.0
- Platform and version: Linux Ubuntu 20.04 64bits kernel 5.4.0-64-generic with all updates installed

#### Expected behavior

Running the AppImage starts the GUI normally

#### Actual behavior

Running the AppImage results in a blank screen

#### Steps to reproduce the behavior

Just run it
```
./ledger-live-desktop-2.20.0-linux-x86_64.AppImage 
```
<img width="1060" alt="Ledger Live Bug" src="https://user-images.githubusercontent.com/149631/106149043-1750ae80-617a-11eb-86d8-fe92761818fb.png">

**Comment by emx** [comment]
It works fine on Ubuntu though, looks like the problem is specific to XFCE-based Xubuntu

**Comment by pscopic** [comment]
Yup. The app is garbage. Running arch it's doing the same thing. It'll just hang.

**Comment by gre** [comment]
@emx Sorry for the inconvenience, we'll investigate to find the root cause of this bug.
you mentioned <2.19 was working fine and 2.20 no longer works for you. What I can see on my side in term of "big" changes is that we have upgraded from `electron 9.2.0 → 11.1.1`. I suspect something broke in https://github.com/electron/electron for your distribution, I tried to find similar complaints on Electron repository but couldn't find.
Is there any other logs apart from the one you shared in the console?

**Comment by LouisLoudogTrottier** [comment]
CentOS is broken since 2.19 also.
Was hoping to get a fix but instead the update deleted my working 2.18.

**Comment by dvwright** [comment]
ledger-live-desktop-2.21.3-linux-x86_64.AppImage is fully working for me on the default gnome desktop environment on a fresh install of Ubuntu 20.04.2.0 LTS

@emx it's not working from within the xfce gui environment? have you tried other desktop GUI environments? (e.g gnome/kde/mate, etc).

@pscopic, @LouisLoudogTrottier what desktop environment are you running?

**Comment by emx** [comment]
@gre I also suspect it's Electron-related (see the error message on my screenshot upon launch)

@dvwright yes I tested it on standard Ubuntu and it worked fine. Didn't try other window managers under Xubuntu, though.

Rather than tinkering with my system, I'm using the 2.19 version that has no issues. Now that there is a 2.21 I might give it a spin to see if the problem has been fixed.

**Comment by LouisLoudogTrottier** [comment]
OMG i forgot about this, Did they fix it? I've stop using Ledger products all together.

**Comment by emx** [comment]
It's still not fixed. I have just tried on latest version (2.29.0) and it is still broken
